cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/CruiseControlRebalanceResponse.java

```diff
@@ -8,28 +8,30 @@
 
 public class CruiseControlRebalanceResponse extends CruiseControlResponse {
 
-    private boolean notEnoughDataForProposal;
-    private boolean proposalIsStillCalculating;
+    private boolean isSufficientDataForProposal;
```

Comment:
as already mentioned above I think we should leave the logic `notEnoughData...` as it is in Cruise Control.